packages/kit-bg/src/migrations/indexedToBucketsMigration/legacyIndexedDb.ts

```
@@ -0,0 +1,183 @@
+          upgrade: (db, oldVersion, newVersion, transaction) => {
+            this._handleDbUpgrade({
+              db,
+              oldVersion,
+              newVersion,
+              transaction,
+            });
+          },
+        },
+      );
+      resolve(legacyDb);
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)